specification/monitor/resource-manager/Microsoft.Insights/stable/2015-04-01/autoscale_API.json

```diff
@@ -442,6 +442,13 @@
           "type": "number",
           "format": "double",
           "description": "the threshold of the metric that triggers the scale action."
+        },
+        "dimensions": {
```

Comment:
Adding a read-write property is deemed as breaking change.

Reply:
Bugfix for missing dimension property in Swagger.